pages/linux/lci.md

```diff
@@ -0,0 +1,16 @@
+# lci
+
+> LOLCODE interpreter written in C.
+> More information: <http://www.lolcode.org>.
```

Comment:
```suggestion
> More information: <https://github.com/justinmeza/lci>.
```
Their site seems to not work properly, it is better to point towards their GitHub I guess.